st3/sublime_lib/show_selection_panel.py

```diff
@@ -10,16 +16,16 @@
 
 
 def show_selection_panel(
-    window,
-    items,
+    window: sublime.Window,
+    items: _Sequence[_ItemType],
     *,
-    flags=0,
-    labels=None,
-    selected=NO_SELECTION,
-    on_select=None,
-    on_cancel=None,
-    on_highlight=None
-):
+    flags: Any = 0,
+    labels: Union[_Sequence[object], Callable[[_ItemType], object]] = None,
```

Comment:
More `Any` here?

Reply:
`object` is correct; we're not doing anything to that value except calling `str`, which is allowed on any value.

Reply:
Well, yeah, but isn't Any the same then? All things in Python are objects.